### PR TITLE
Rename compilation warning

### PR DESCRIPTION
**Pull request by sprisha**
#### What type of PR is this?

 Feature Improvement 

#### What does this PR do / why is it needed ?

 add's source information to add warnings + reframes the test example for duplicate columns in a table

### CODE REVIEW

File: legend-engine-xt-relationalStore-grammar/src/test/java/org/finos/legend/engine/language/pure/compiler/test/TestRelationalCompilationFromGrammar.java [hunk 1]
```diff
@@ -266,12 +267,15 @@ public void testRelationalDatabaseFail()
...
+            warnings.add("COMPILATION error: Duplicate column definitions [FIRSTNAME, LASTNAME] in table: personTable");
+            warnings.add("COMPILATION error: Duplicate column definitions [ADDRESSID, LEGALNAME] in table: firmTable");
+            warnings.add("COMPILATION error: Duplicate column definitions [LEGALNAME] in table: otherFirmTable");
+            PureModel dbIncModel = test(DB_DUP_INC, null, warnings).getTwo();
         }
         catch (AssertionError e)
         {
-            Assert.assertEquals("expected no warnings but found [COMPILATION error: Duplicate column definitions [FIRSTNAME, LASTNAME] in table: personTable, COMPILATION error: Duplicate column definitions [ADDRESSID, LEGALNAME] in table: firmTable, COMPILATION error: Duplicate column definitions [LEGALNAME] in table: otherFirmTable]", e.getMessage());
-        }
+            Assert.assertEquals("expected:<[COMPILATION error: Duplicate column definitions [FIRSTNAME, LASTNAME] in table: personTable, COMPILATION error: Duplicate column definitions [ADDRESSID, LEGALNAME] in table: firmTable, COMPILATION error: Duplicate column definitions [LEGALNAME] in table: otherFirmTable]>", e.getMessage());
```
Comment (Yasirmod17) [inline legend-engine-xt-relationalStore-grammar/src/test/java/org/finos/legend/engine/language/pure/compiler/test/TestRelationalCompilationFromGrammar.java:278]: missing a curly brace after this?
